configure.ac

```diff
@@ -3713,6 +3713,10 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
   [AC_MSG_RESULT(no)
 ])
 
+if test "x${ac_cv_func_clock_gettime}" = xno; then
+  AC_MSG_ERROR([could not find clock_gettime (see https://bugs.python.org/issue40174)])
```

Comment:
This change is wrong.

clock_gettime is not required on all platforms: https://bugs.python.org/issue40174#msg365775

Reply:
Well, spotted, I'll fix that!

"Where is the monster? Behind the rabbit?"